docs/grammar.txt

```diff
@@ -72,8 +75,11 @@ VariableDefinition = ('var' IdentifierList | VariableDeclaration) ( '=' Expressi
 IdentifierList = '(' ( Identifier? ',' )* Identifier? ')'
 
 // Precedence by order (see github.com/ethereum/solidity/pull/732)
-Expression =
-  ( Expression ('++' | '--') | FunctionCall | IndexAccess | MemberAccess | '(' Expression ')' )
+Expression
+  = Expression ('++' | '--')
```

Comment:
Whoops, yes. My ANTLR grammar had `'new' Expression` as part of the list. Didn't notice when I migrated the changes to grammar.txt